k9mail/src/main/java/com/fsck/k9/activity/setup/outgoing/OutgoingPresenter.java

```diff
@@ -57,37 +55,40 @@ public void setAccount(String accountUuid) {
                 return;
             }
         } catch (URISyntaxException e) {
-            Timber.e(e, "Account retrieved from intent doesn't have valid store URI");
-            return;
+            view.onAccountLoadFailure(e);
```

Comment:
The way to handle this error is from original codes. But is it ok to just show a toast and do nothing else like finishing the whole activity?